test/test_core.py

```diff
@@ -3958,6 +3958,51 @@ def test_dlfcn_rtld_local(self):
 
     self.do_runf('main.c', 'main\nfunc_a\nfunc_sub\nfunc_b\nfunc_sub\ndone\n')
 
+  def test_dlfcn_preload(self):
```

Comment:
Adding the `@needs_dylink` decorator should fix the wasm2js test failure.

Reply:
Yep, done.